tests/fsharp/typecheck/sigs/neg105.bsl

```diff
@@ -0,0 +1,10 @@
+
+typecheck error FS0019: This constructor is missing arguments; 2 are expected. Patterns must unpack them, e.g., 'Rect (width, height)'
```

Comment:
Something is wrong here... the source file path isn't appearing before the error and I have no clue why. Anyone know?